tests-legacy/phpunit.xml

```diff
@@ -1,5 +1,4 @@
 <phpunit bootstrap="bootstrap.php"
-         stopOnFailure="true"
```

Comment:
I think this setting was introduced in order not to waste GitHub Action runner time.

But it would make sense to have the full CI running, if we are not short of resources. Wdyt @PierreRambaud @eternoendless ?

Reply:
The StopOnFailure has to be set to false when running under CI, our unit tests should rely on other unit tests, so we can continue to see if other tests are failing. It's not like UI tests which if one is failing, you already know they others are going to fail ^^

Reply:
so please rebase and merge, guys

Reply:
Do you really think I will do it by talking like that? :thinking:

Reply:
yes, to save time on every side :)